shared/js/content-scripts/block-cookie.js

```diff
@@ -0,0 +1,143 @@
+/* global Cookie */
+// Set up 3rd party cookie blocker
+(function cookieBlocker () {
+    // don't inject into non-HTML documents (such as XML documents)
+    // but do inject into XHTML documents
+    if (document instanceof HTMLDocument === false && (
+        document instanceof XMLDocument === false ||
+        document.createElement('div') instanceof HTMLDivElement === false
```

Comment:
The page could game this to return a different prototype.

Reply:
This check is run in content-script scope, where that cannot happen, right?

Reply:
Yeah sorry I thought this was injected :)